python/lsst/pipe/tasks/calibrate.py

```diff
@@ -344,19 +358,22 @@ def DebugInfo(name):
     ConfigClass = CalibrateConfig
     _DefaultName = "calibrate"
 
-    def __init__(self, **kwargs):
+    def __init__(self, schema=None, **kwargs):
         """!
         Create the calibration task
 
-        \param **kwargs keyword arguments to be passed to lsst.pipe.base.task.Task.__init__
+        \param[in] schema    input SourceTable schema
+        \param     **kwargs  keyword arguments to be passed to lsst.pipe.base.task.Task.__init__
         """
         pipeBase.Task.__init__(self, **kwargs)
 
         # the calibrate Source Catalog is divided into two catalogs to allow measurement to be run twice
         # schema1 contains everything except what is added by the second measurement task.
         # Before the second measurement task is run, self.schemaMapper transforms the sources into
-        # the final output schema, at the same time renaming the measurement fields to "initial_" 
-        self.schema1 = afwTable.SourceTable.makeMinimalSchema()
+        # the final output schema, at the same time renaming the measurement fields to "initial_"
+        if schema is None:
+            schema = afwTable.SourceTable.makeMinimalSchema()
+        self.schema1 = schema
```

Comment:
I'd suggest keeping it, but adding a note to the commit message noting that it's included. Not perfect, but better than dropping it or silently including it.